### PR TITLE
HOTFIX 63.4

### PR DESCRIPTION
**Pull request by ZappoMan**
Fixes for some things that were causing crashing.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7127/app_12383_PR.apk

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -2220,25 +2230,23 @@ void Application::initializeGL() {
...
+    DeadlockWatchdogThread::withPause([&] {
+        // Set up the render engine
+        render::CullFunctor cullFunctor = LODManager::shouldRender;
+        static const QString RENDER_FORWARD = "HIFI_RENDER_FORWARD";
+        bool isDeferred = !QProcessEnvironment::systemEnvironment().contains(RENDER_FORWARD);
+        _renderEngine->addJob<UpdateSceneTask>("UpdateScene");
+#ifndef Q_OS_ANDROID
+        _renderEngine->addJob<SecondaryCameraRenderTask>("SecondaryCameraJob", cullFunctor, isDeferred);
 #endif
+        _renderEngine->addJob<RenderViewTask>("RenderMainView", cullFunctor, isDeferred, render::ItemKey::TAG_BITS_0, render::ItemKey::TAG_BITS_0);
```
Comment (samcake) [inline interface/src/Application.cpp]: THis is not in sync with the RC63.x branch (it s in rc64)

Comment (ZappoMan) [reply interface/src/Application.cpp]: this was branched off of stable... so it definitely IS what is currently available.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7130/app_12383_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7130/HighFidelity-Beta-PR12383-f6fbac25b0477b8a9f6d7a940e4eda9742bc1b7d.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7130/HighFidelity-Beta-PR12383-f6fbac25b0477b8a9f6d7a940e4eda9742bc1b7d.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7132/app_12383_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7132/HighFidelity-Beta-PR12383-bc51f3643d87c85e44f658af91daa87737f1d69b.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7132/HighFidelity-Beta-PR12383-bc51f3643d87c85e44f658af91daa87737f1d69b.exe

**Comment by Qliemillar** [comment]
Testing complete - PASS @conklin94122

**Comment by conklin94122** [comment]
Test plan - https://highfidelity.testrail.net/index.php?/plans/view/2854

**Pull request merged**